### PR TITLE
Adjust doc MNIST learning rate

### PR DESCRIPTION
**Pull request by pvaneck**
The gradient descent learning rate specified in the MNIST beginner
tutorial should be 0.5 instead of 0.05 to stay consistent with the
actual code in the repo.

A learning rate of .05 will actually decrease accuracy.

Code in repo: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/examples/tutorials/mnist/mnist_softmax.py#L59

### CODE REVIEW

**Review by yifeif — APPROVED** [review]
Nice catch, thanks!

**Comment by foxish** [comment]
The documentation on the [website](https://www.tensorflow.org/get_started/mnist/beginners) still says 0.05. Some generation script needs to be run to update that?

**Comment by yifeif** [comment]
Looks like the code block on the website was somehow not updated with this change, but the second instance that is not in the code block got updated. @MarkDaoust have you run into this before?

**Comment by MarkDaoust** [comment]
It looks like there's an intermediate version of this page where that was the actual content, then someone updated the comment in the doc to match the code in the doc...

The root docs (that you link to) are only updated with releases.

But this fix will be in [versions/master](https://www.tensorflow.org/versions/master/get_started/mnist/beginners) next time it gets published.
